ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-multi.yaml

```diff
@@ -0,0 +1,98 @@
+base_images:
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.17"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: control-plane-10nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: m7g.metal
+      ADDITIONAL_WORKERS: "5"
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "5"
+      COMPUTE_NODE_TYPE: m5zn.metal
+      CONTROL_PLANE_INSTANCE_TYPE: m5zn.metal
+      OCP_ARCH: amd64
+    test:
+    - ref: ipi-install-heterogeneous
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-aws
+  timeout: 6h0m0s
+- always_run: false
+  as: data-path-4nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: m7g.metal
+      ADDITIONAL_WORKERS: "2"
```

Comment:
@paigerube14 I think 3 worker nodes should be enough, and this is the config we have in our [config google sheet](https://docs.google.com/spreadsheets/d/1DdFiJkCMA4c35WQT2SWXbdiHeCCcAZYjnv6wNpsEhIA/edit?gid=193707991#gid=193707991) now. So let's update this to ADDITIONAL_WORKERS: "1". 4.16 job can be updated too. Tnx! 
@liqcui Liquan, let me know if you have any comments.

Reply:
I copied what was on 4.16. I'll update both 4.16 and 4.17 with 1 additional worker and 2 base workers